packages/mock-server/src/api/mutations.ts

```diff
@@ -44,9 +45,20 @@ export const insert = {
 
     return { ...createdInvoice, __typename: 'InvoiceNode' };
   },
-  invoiceLine: (
-    invoiceLine: InsertOutboundShipmentLineInput
-  ): InsertOutboundShipmentLineInput => {
+
```

Comment:
whole mock server /worker are getting pretty messy. Not sure what the plan is - I suppose keeping it around for awhile longer so might be worth a couple hour cleanup pass soon 🤔